apps/etterlatte-saksbehandling-ui/client/src/shared/statusbar/Statusbar.tsx

```diff
@@ -29,32 +29,38 @@ export const StatusBar = ({ result }: { result: Result<IPersonResult> }) => {
     return GenderList.male
   }
 
-  return (
-    <StatusBarWrapper>
-      {isPending(result) && (
+  return mapApiResult(
+    result,
+    <PersonSkeleton />,
+    () => null,
```

Comment:
Hvorfor null her?

Reply:
Synes det var ryddigere å fjerne person-/statusbar helt i tilfeller hvor person ikke kan hentes ut, men er åpen for alternativer 🤷‍♂️